include/alpaka/vec/Vec.hpp

```diff
@@ -402,6 +404,34 @@ namespace alpaka
             return r;
         }
 
+        //! \return The element-wise minimum of one or more vectors.
+        ALPAKA_NO_HOST_ACC_WARNING
+        template<typename... Vecs, typename = std::enable_if_t<boost::mp11::mp_same<Vec, Vecs...>::value>>
```

Comment:
We have decided against the integration of Boost.Mp11 a while back. From what I can see you need it for `mp_same`. Shouldn't it be possible to use C++17 fold expressions here?

```suggestion
        template<typename... Vecs, typename = std::enable_if_t<std::is_same_v<Vec, Vecs> && ...>>
```

CC @bernhardmgruber

Reply:
Let me rephrase: We don't have consensus yet to allow the use of Boost.Mp11. The relevant discussion is here #1296.

In this particular instance, the fold expression suggested by Jan should work equally well!

Reply:
OK, I see.

Thanks, the fold expression does seem to work well (it only needs a couple of `(...)`).